app/controllers/search_controller.rb

```diff
@@ -33,7 +33,7 @@ def extract_filters(response)
     aggs = response&.data&.search&.to_h&.dig('aggregations')
     return if aggs.blank?
 
-    aggs.select { |_, agg_values| agg_values.present? }
+    aggs.select { |_, agg_values| agg_values.present? }.transform_keys { |key| (key.dup << 'Filter').to_sym }
```

Comment:
Does this feel like a workaround to a deficiency in our API? Or is this something only our UI is likely to encounter?

It feels like we should be returning our aggregations using the same terminology as we use to apply them. So instead of returning `source` we should return that aggregation as `sourceFilter` so it is clearer how to use the data. Thoughts?

Reply:
Possibly. I hate this code, and manipulating the response this much in the UI certainly _feels_ like a workaround, but I don't know if I'm convinced we should change our aggregation names. Since they're not filter/filters aggregations, naming them as such might be confusing to other consumers. Less confusing than having different names for the corresponding aggs, though? I'm not sure.